tests/cloner_test.go

```diff
@@ -108,6 +108,34 @@ var _ = Describe("all clone tests", func() {
 			completeClone(f, f.Namespace, targetPvc, diskImagePath, sourceMD5, sourcePvcDiskGroup)
 		})
 
+		It("[test_id:XXXX]Should clone imported data from SourceRef PVC DataSource", func() {
```

Comment:
It("[test_id:6784]Should clone imported data from SourceRef PVC DataSource", func() {